.bumpversion.cfg

```diff
@@ -21,6 +21,6 @@ values =
 [bumpversion:file:code.json]
 
 [bumpversion:file:package.json]
-
-[bumpversion:file:package-lock.json]
+search = "version": "{current_version}"
```

Comment:
Wouldn't this still potentially be problematic after releases? For example, if a javascript package was also at version 0.4.0 or something similar.

I'm wondering if a multiline specification of the search string would give better specificity?

Reply:
That line applies to package.json. I have removed package-lock.json from bumpversion. In package.json, only the package meta data has version: {version}

Reply:
Ah. Yes. Should have looked more closely.